tests/test_docs/test_standalone_transaction/test_standalone_transaction.py

```diff
@@ -37,7 +37,6 @@
 test_logger = logging.getLogger(__name__)
 
 
-@pytest.mark.integration(reruns=MAX_FLAKY_RERUNS_INTEGRATION)
```

Comment:
the mark should be only to the end-to-end test.

Reply:
this is an end-to-end test

Reply:
I mean: instead of marking the entire test class, let's just mark the end-to-end test.